torch/_functorch/autograd_function.py

```diff
@@ -267,16 +291,14 @@ def custom_function_call_vmap(interpreter, autograd_function, *operands):
             return custom_function_call(autograd_function, *operands)
 
     with interpreter.lower():
-        unwrapped_output, out_dims = autograd_function.vmap(info, in_dims, *unwrapped_operands)
+        result = autograd_function.vmap(info, in_dims, *unwrapped_operands)
+    validate_vmap_returns_two(result)
```

Comment:
nit: I think we might as well fully describe it here, then a reader won't even have to read the function to figure out what it does - "validate_vmap_returns_tuple_of_two_elements".